app/src/main/java/org/schabi/newpipe/player/helper/AudioReactor.java

```diff
@@ -86,6 +87,14 @@ private boolean shouldBuildFocusRequest() {
         return Build.VERSION.SDK_INT >= Build.VERSION_CODES.O;
     }
 
+    public void registerMediaButtonEventReceiver(ComponentName componentName) {
+        audioManager.registerMediaButtonEventReceiver(componentName);
```

Comment:
We should add a check to disable this (and also log an error) after Oreo, since we don't know if this will be removed in the future.